src/_preflights/tw-reset.js

```diff
@@ -35,12 +35,14 @@ html {
 
 /*
 1. Remove the margin in all browsers.
-2. Inherit line-height from 'html' so users can set them as a class directly on the 'html' element.
+2. Base font size.
+3. Base line height.
 */
 
 body {
   margin: 0; /* 1 */
-  line-height: inherit; /* 2 */
+  font-size:var(--w-font-size-m); /* 2 */ 
+  line-height:var(--w-line-height-m); /* 3 */
 }
```

Comment:
In the [Typography task](https://trello.com/c/ONEZZuP7/213-add-typography-tokens) we were supposed to style the `body` with the following:
```
body {
    /* Set the default site wide bg-color, text-colour, and default font-size with line-height */
    @apply bg text text-m;
   }
```
I haven't come that far but it looks like you covered a similar issue here 😊